src/commons/assessmentWorkspace/AssessmentWorkspace.tsx

```diff
@@ -516,7 +516,7 @@ const AssessmentWorkspace: React.FC<AssessmentWorkspaceProps> = props => {
         afterDynamicTabs: []
       },
       onChange: onChangeTabs,
-      workspaceLocation: 'assessment'
+      workspaceLocation: workspaceLocation
```

Comment:
```suggestion
      workspaceLocation
```

Reply:
Hmm I don't think this is really needed (also separating it this way helps when ctrl-clicking). Perhaps we can take another look at this after the Workspace and MobileWorkspace props have been refactored (#2518).